libs/novui/src/recipes/input-editor-widget.recipe.ts

```diff
@@ -0,0 +1,50 @@
+import { defineSlotRecipe } from '@pandacss/dev';
+import { RichTextEditorStylesNames } from '@mantine/tiptap';
+import { INPUT_RECIPE } from './input.recipe';
+
+// full enumeration of the component library's slots
+const SLOTS: RichTextEditorStylesNames[] = [
+  'linkEditorSave',
+  'linkEditorDropdown',
+  'root',
+  'content',
+  'typographyStylesProvider',
+  'control',
+  'controlsGroup',
+  'toolbar',
+  'linkEditor',
+  'linkEditorInput',
+  'linkEditorExternalControl',
+];
+export const INPUT_EDITOR_WIDGET_RECIPE = defineSlotRecipe({
+  className: 'inputEditorWidget',
+  jsx: ['InputEditorWidget'],
+  slots: SLOTS,
+  base: {
+    root: {
+      ...INPUT_RECIPE.base.input,
+      _groupError: {
+        borderColor: 'input.border.error',
+      },
+    },
+    content: {
+      bg: 'inherit',
+
+      // override tiptap styles
+      '& .tiptap': {
+        p: '0',
+      },
+      // appearance of autocomplete suggestion after being selected and added inline to the input
+      '& .suggestion': {
+        color: 'variable.text',
+        borderRadius: 'xs',
+        background: 'variable.surface',
+        p: '25',
+        lineHeight: '125',
+        borderColor: 'variable.border',
+        border: 'solid',
+      },
+    },
+  },
+  staticCss: ['*'],
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
🗒 **note (non-blocking):** This tells Panda to generate styles for all variants and slots, regardless of which ones are actually used in code. When using `defineSlotRecipe`, Panda normally only generates classes and styles for recipes that the static checker detects are used in code.‏